control_api.go

```diff
@@ -76,6 +78,10 @@ func (c *ControlAPI) Listen() error {
 		}
 	}
 
+	if err := os.Chmod(c.socket, os.ModeSocket|SocketPerm); err != nil {
```

Comment:
`os.ModeSocket` is not even needed here, as os.Chmod will only use the `FileInfo.Perm()` bits of the argument.

Just a note, it's OK to leave it there for clarity, no need to change it.